release-notes.py

```diff
@@ -62,11 +72,30 @@
     print("Writing to RELEASE.md OK!")
     release_suppfile = open("release_supp.data", "a")
     try:
-        release_suppfile.write(app_ver_num+"\n") == 0
+        release_suppfile.write(app_ver_numbr+"\n") == 0
     except IOError:
         print("Error writing to support file!")
     else:
         print("Writing to support file OK!")
     release_suppfile.close()
 
 releasemd.close()
+
+message_text = ""
+if xml_message:
+    if app_use_tckt == "true":
+        print("Please write ticket and press Enter")
+        ticket_txt = input()
+        message_text = app_msg+app_ver_numbr+", "+ticket_txt
+    elif app_use_tckt == "false":
+        message_text = app_msg+app_ver_numbr
+
+    subprocess.run(["git", "add", "RELEASE.md"])
+    subprocess.run(["git", "add", "release_supp.data"])
+    subprocess.run(["git", "commit", "-m", message_text])
+    subprocess.run(["git", "push"])
+
+else:
+    print("No message template defined in pom.xml file!")
```

Comment:
Maybe navigate user how to configure it, but not necessary of course. I would add something like "follow ## How to run? in [README](https://github.com/smilingwords/maven-release-notes/blob/main/README.md))